### DISCUSSION TITLE
[Feature request] Toggle timer on/off button on main menu

### DISCUSSION THREAD

**Discussion by rieje** [arendst/Tasmota#21882] [Ideas]
I use timers often and I think it would be really to have a `Toggle Timer On/Off` button on the main menu. I have use cases where I need to toggle timer on/off as a one-time thing and navigating menus to set this seems too involved for a feature used frequently.

Another useful feature would to allow turning off the upcoming timer (that would trigger the switch on or off) as a one-time thing--once that upcoming timer passes, switch the timer on again.

**Comment by Jason2866** [comment]
Thx for sharing your ideas. A additional button will probably not be added to the main menu to switch on off timers. This clutters the clear menu structure. In other use cases other functions in a sub menu would be nice to see in main menu. Will end every function is visible at the main page.

The second. It is a niche feature.

**Comment by sfromis** [comment]
If you really want a front page toggle for such a niche purpose (remember that the Tasmota web UI is not even meant for "daily operation"), you could crate a virtual Relay (configured on an unused pin), and then have rules to react to that, and issue the command you like.
